test/framework/integration_framework/integration_test_framework.cpp

```diff
@@ -269,27 +269,26 @@ namespace integration_framework {
   };
 
   IntegrationTestFramework &IntegrationTestFramework::sendTx(
-      const shared_model::proto::Transaction &tx, std::set<TxStatus> statuses) {
+      const shared_model::proto::Transaction &tx,
+      std::function<
+          void(const std::vector<shared_model::proto::TransactionResponse> &)>
```

Comment:
Can we change to interface::TransactionResponse?

Reply:
only if it `std::vector<std::shared_ptr<iface::TxResponse>>`

Reply:
Though, it's corresponds to ITF interface, so it looks fine anyway